packages/api/endpoints/async-operations.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
```

Comment:
Functions in this file need to be documented.

Reply:
Done.